dali/python/nvidia/dali/pipeline.py

```diff
@@ -1583,7 +1583,7 @@ def deserialize_and_build(self, serialized_pipeline):
         self._deserialized = True
 
     def save_graph_to_dot_file(
-        self, filename, show_tensors=False, show_ids=False, use_colors=False
+        self, filename, *, show_tensors=False, show_ids=None, use_colors=False
```

Comment:
This `*` is a breaking change!

Reply:
I can document it as such, but this function is for debugging only - we neither document, demonstrate or test it.

Reply:
We do, perhaps unintentionally https://docs.nvidia.com/deeplearning/dali/user-guide/docs/pipeline.html#nvidia.dali.Pipeline.save_graph_to_dot_file

Reply:
....we still don't use it and don't test it.